SPECS/ncurses.spec

```diff
@@ -283,13 +287,864 @@ xz NEWS
 %{_libdir}/lib*.a
 
 %changelog
-* Wed Apr 10 2024 Gerald Elder-Vass <gerald.elder-vass@cloud.com> - 6.4-3
-- CA-391276: Revert previous changes due to regression
+* Wed Mar  5 2025 Yann Dirson <yann.dirson@valtes.tech> - 6.4-5.20240309
+- sync with XS 6.4-4
+- unlisted XS changes:
+  - Upgrade upstream revision to 20240309
+  - Build and run a test program to verify there are no conflicts in lib versions
+- XS changelog:
+  * Wed Aug 14 2024 Gerald Elder-Vass <gerald.elder-vass@cloud.com> - 6.4-4
+  - CP-48941: Update ncurses to MONOTONIC
+
+* Wed Mar  5 2025 Yann Dirson <yann.dirson@valtes.tech> - 6.4-4.20230114
+- restore Fedora changelog and versionning, while keeping other XS-ification
+- XS changelog:
+  * Wed Apr 10 2024 Gerald Elder-Vass <gerald.elder-vass@cloud.com> - 6.4-3
+  - CA-391276: Revert previous changes due to regression
+  * Wed Mar 13 2024 Gerald Elder-Vass <gerald.elder-vass@cloud.com> - 6.4-2
+  - CA-369899: Update to verion 20240309
+  - CA-369899: Use MONOTONIC time instead of REALTIME
+  * Tue Jul 11 2023 Tim Smith <tim.smith@citrix.com> - 6.4-1
+  - First imported release
```

Comment:
There will be just one build so no need for two separate changelog entries

Reply:
After today's talk, we're OK to keep both?
Maybe suffix the first one wih `~unreleased`?

Reply:
Ok for me.

Reply:
I still don't really get the merge commit in the PR, but I can live with it :)